README.md

```diff
@@ -380,6 +380,8 @@ make style && make quality
 └── tests           <- Unit tests
 ```
 
+## Related work
+[jxpress/seifit-pytorch-lightning](https://github.com/jxpress/seifit-pytorch-lightning) - A repository of PyTorch Lightning implementations of SetFit.
```

Comment:
small suggestion to fix a typo and create a bullet list :)
```suggestion

* [jxpress/setfit-pytorch-lightning](https://github.com/jxpress/setfit-pytorch-lightning) - A PyTorch Lightning implementation of SetFit.
```

Reply:
Hi @lewtun 
thank you for approving my PR and suggestion!

I fixed it as you suggested me.

thank you for kind reply :)

Yongtae